src/server_manager/install_scripts/install_server.sh

```diff
@@ -158,7 +158,10 @@ function fetch() {
 }
 
 function install_docker() {
+  umask 0022
+  (chmod a+r /usr/share/keyrings/docker-archive-keyring.gpg 2>/dev/null || true)
```

Comment:
Shouldn't we fail the install if this fails?
Also, why are you redirecting the output to /dev/null?

Reply:
chmod is for already created file if the script it ran not the first time.
it will fail if this is the first run, that' why we need true.